packages/ui/src/components/DropdownLabels.svelte

```diff
@@ -42,6 +42,8 @@
   export let minW0 = true
   export let disabled: boolean = false
 
+  export let withSearch: boolean = true
```

Comment:
Why not use the enableSearch?

Reply:
To make it consistent with `Dropdown` which has `withSearch` prop :)
Renamed to `enableSearch` here anyway, thanks!

Reply:
In this case we need to make it consistent in all places.